collections/_topics/klimaticka-zmena.md

```diff
@@ -56,7 +56,7 @@ dashboard:
   data:
   - value:     "**20** cm"
     subtitle:  "do roku 2018"
-  - value:     "**80–150** cm"
+  - value:     "**80 – 150** cm"
```

Comment:
Jsi si jistá touto úpravou?

V češtině je to správně bez mezer, viz https://prirucka.ujc.cas.cz/?ref=164&id=165.
Ve slovenštině samozřejmě nevím, ale proto se ptám :)

Reply:
hej, my to máme inak v slovenčine, viď [môj obľúbený zdroj na kopírovanie pomlčky](https://sk.wikipedia.org/wiki/Pomoc:Typografia#Poml%C4%8Dka_(%E2%80%93))

Reply:
🤯